From e6f379c9193bb8a0c34e182139f33b11b70da18b Mon Sep 17 00:00:00 2001
From: Alison Wang <alison.wang@nxp.com>
Date: Thu, 19 Jul 2018 17:32:11 +0800
Subject: [PATCH 273/706] drm: hdp: Remove mode setting in the thread for HPD

This patch removes  mode setting in the thread for HPD. The
corresponding time for timeout is increased.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
(cherry picked from commit a906133d468f89266041e9e8b58c37b0ea5dba68)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-hdp.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-hdp.c b/drivers/gpu/drm/imx/hdp/imx-hdp.c
index 754ab08f7eba..0cb18b014d06 100644
--- a/drivers/gpu/drm/imx/hdp/imx-hdp.c
+++ b/drivers/gpu/drm/imx/hdp/imx-hdp.c
@@ -865,7 +865,6 @@ static int hpd_det_worker(void *_dp)
 				imx_hdp_cable_plugin(hdp);
 				hdp->cable_state = true;
 				drm_kms_helper_hotplug_event(hdp->connector.dev);
-				imx_hdp_mode_setup(hdp, &hdp->video.cur_mode);
 			} else if (HPDevents & 0x2) {
 				hdp->cable_state = false;
 				imx_hdp_cable_plugout(hdp);
@@ -877,7 +876,7 @@ static int hpd_det_worker(void *_dp)
 		} else if (evt & 0xf)
 			printk(KERN_DEBUG "evt=0x%x\n", evt);
 
-		schedule_timeout_idle(1);
+		schedule_timeout_idle(100000);
 	}
 
 	return 0;
-- 
2.17.1

